homeassistant/components/suez_water/quality_scale.yaml

```diff
@@ -0,0 +1,88 @@
+rules:
+  # Bronze
+  config-flow: todo
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data:
+    status: todo
+    comment: coordinator is created during setup, should be stored in runtime_data
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: no subscription to api
+  dependency-transparency: done
```

Comment:
Can you please detach the dependency repo from the original fork btw?

Reply:
Done, forked has been detached by Github team.